Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -290,6 +290,36 @@ static RefPtr<ImageBuffer> snapshotNodeVisualOverflowClippedToViewport(LocalFram
     return result;
 }
 
+static ExceptionOr<void> forEachElementInPaintOrder(const std::function<ExceptionOr<void>(Element&)>& function, RenderLayer& layer)
```

Comment:
Might be worth adding a comment (or changing the name) to note that this only visits Elements that have a RenderLayer associated with them (which is sufficient for view-transitions, since `view-transition-name` forces creation of a layer).